### DISCUSSION TITLE
Compare with Apache Drill

### DISCUSSION THREAD

**Discussion by dpy1123** [databendlabs/databend#8873] [Q&A] [answered]
Hi team! I'm new to this project, and I'm curious about the difference with [apache drill](https://drill.apache.org/) project, from both architecture perspective and performance perspective.
Thx

**Comment by sundy-li** [comment, marked as answer]
Hi, I'm not familiar with Drill, but it seems to be a distributed query engine based on standard storage systems (HDFS, MongoDB etc).

Databend is not only a query engine but a fully functional data warehouse (elastic query +  elastic meta service + elastic storage)
